src/mobile-token/DebugTokenServerAddress.tsx

```diff
@@ -0,0 +1,58 @@
+import {Button} from '@atb/components/button';
+import {ThemeText} from '@atb/components/text';
+import {storage} from '@atb/storage';
+import {StyleSheet} from '@atb/theme';
+import {useEffect, useState} from 'react';
+import {TextInput, View} from 'react-native';
+
+export const DebugTokenServerAddress = () => {
+  const [ipAddress, setIpAddress] = useState<string>('');
+
+  const saveDebugIpAddress = async (ipAddress: string) => {
+    if (ipAddress.length > 0) {
+      const fullAddress = `http://${ipAddress}:8080`;
```

Comment:
Not quite sure what's useful to test in practice, but should we have the input be the whole address instead of just the IP? If it's useful to e.g. for point a prod device to staging servers, where we don't have access to a server running on a local IP.

Reply:
We usually save that into the .env file and read from it

Reply:
We can change it to a whole IP (probably with a `http://` prefix added) 👍🏼 

Also, pointing a prod device to staging server won't work, because the tokens on Prod are different from tokens from Staging.

This is just to make it easier for testing on staging, either for reproducing a bug, or testing/developing new functionality. For example: we want to replicate a bug with creating tokens, we can just point the token server to local, instead of changing some parameters from the code to allow debug. These changes might be minor, but the added overhead of having to modify the server is something that we could have avoided with this menu.

Reply:
Refactored to allow full address